### PR TITLE
[params] Fixes events/imports_controller.rb

### PR DESCRIPTION
**Pull request by nidhi-soni1104**
**What this PR does / why we need it:**
Fixes usage of params in the imports controller

### CODE REVIEW

File: app/controllers/master/events/imports_controller.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def create
   private
 
   def check_shared_secret
-    head(403) unless params[:secret] == Events.shared_secret
+    permitted_params = params.permit(%i[secret]).to_h
```
Comment (hallelujah) [inline app/controllers/master/events/imports_controller.rb:16]: Why `to_h`?

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: app/controllers/master/events/imports_controller.rb (hunk 1 above)
Comment (nidhi-soni1104) [reply app/controllers/master/events/imports_controller.rb:16]: @hallelujah we (@jsmadis  and @josemigallas )discussed in the call that its safe to convert to hash , so if we permit we convert it to hash.

Comment (nidhi-soni1104) [reply app/controllers/master/events/imports_controller.rb]: @hallelujah PR resolved

**Comment by akostadinov** [comment]
Just curious, which test failed without this change? I wonder if it is needed.

**Review by hallelujah — APPROVED** [review]

**Comment by nidhi-soni1104** [comment]
@akostadinov Not sure about the test, these are few files listed by Jakub which caused faliures due to params unpermitted issues so fixing those listed files.

**Pull request merged**